sdk/mixedreality/Azure.MixedReality.RemoteRendering/src/Azure.MixedReality.RemoteRendering.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyTitle>Microsoft Azure Mixed Reality ARR Client</AssemblyTitle>
+    <Version>1.0.0-preview.1</Version>
+    <PackageTags>Azure MixedReality</PackageTags>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Authors>Microsoft</Authors>
+    <PackageProjectUrl>https://azure.microsoft.com/services/remote-rendering/</PackageProjectUrl>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <Nullable>enable</Nullable>
+    <DefineConstants>$(DefineConstants);AZURE_NULLABLE</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.MixedReality.Authentication" VersionOverride="1.0.0-alpha.20210203.1" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
You shouldn't need a STJ package reference as Azure.Core brings it into the closure.